cli/lsp/language_server.rs

```diff
@@ -97,7 +94,7 @@ pub(crate) struct Inner {
   diagnostics_server: diagnostics::DiagnosticsServer,
   /// The collection of documents that the server is currently handling, either
   /// on disk or "open" within the client.
-  documents: Documents,
+  pub(crate) documents: Documents,
```

Comment:
These `pub(crate)`'s are temporary. They'll be removed in a future PR once nothing depends on this struct.